R/metab.bookkeep.R

```diff
@@ -73,7 +73,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
 		warning("datetime not found, inferring sampling frequency from # of observations") # issue a warning (note checks in addNAs)
 		# NOTE: because of the checks in addNA's, it is unlikely a user would receive this warning via metab()
 		# warning will only be seen through direct use of metab.bookkeep when datettime is not supplied
-		freq <- nobs
+		freq <- nobs - 1
```

Comment:
This issue is confusing me. I didn't dig into the code, but I had always assumed the time series would start at midnight and proceed up to and including the sample just _before_ the next midnight. So start at 0:00, 01:00, ... then end 23:00 for hourly. So 24 samples, 23 differences.

So for 1 day of data, a user _should_ only supply 1 midnight value, right? I was surprised by comments suggesting otherwise.